GDJS/Runtime/runtimegame.ts

```diff
@@ -989,6 +1005,10 @@ namespace gdjs {
      * time.
      */
     _setupSessionMetrics() {
+      if (this._isDisposed) {
+        return;
+      }
```

Comment:
Remove this check, and clear the interval instead.

Reply:
done